mbid_mapping/mapping/custom_sorts.py

```diff
@@ -106,6 +114,37 @@
 ]
 
 
+def get_combined_release_group_types_sort():
```

Comment:
This has got to be one of the weirder functions you've written in your life, I am sure. :)

Reply:
Haha yes.